### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by MaZe3D** [x-radeon/unraid-powershell#1]
Polish it and get it into the Unraid Plugin Repository, to make auto-updates easier and the plugin more accessible.

**Comment by x-radeon** [comment]
Yeah I'll have to look at that, I'm not sure how I get into the Unraid Plugin repo.

I think the plugin is fairly decent. XD I think I'm doing some hacks that I could just do if I knew how to actually write an Unraid plugin.

Updates should happen automatically. As soon as the plugin version increments, if you hit "check for updates" in the plugin tab, it will detect that an update is needed. If you use another plugin to auto update your plugins, it will get updated as well.

**Comment by x-radeon** [comment]
I created an Unraid community post here for the plugin https://forums.unraid.net/topic/135932-plugin-powershell/

**Comment by MaZe3D** [comment]
With polishing I meant integrating a Propper logo. The community post is of course a great step in the right direction.

**Comment by itsalljustdata** [comment]
Looking at the [Unraid forums](https://forums.unraid.net/topic/52623-plugin-system-documentation/), it looks like this is the sort of syntax for getting an icon in there

``` XML
<FILE Name="/boot/config/plugins/PowerShell/icon.png">
   <URL>https://raw.githubusercontent.com/x-radeon/unraid-powershell/main/icon.png</URL>
</FILE>

<FILE Name="/usr/local/emhttp/plugins/PowerShell/icon.png">
   <LOCAL>/boot/config/plugins/PowerShell/icon.png</LOCAL>
</FILE>
```

of course, me being me, i'd put a load more variables as ENTITY values to avoid repeating a load of that URL etc.. (but i've already got a PR on the go!)
